frontend/src/themes/dark-theme.ts

```diff
@@ -617,5 +617,12 @@ export default createTheme({
                 }),
             },
         },
+
+        // Default Icon to Material Symbols Outlined font
```

Comment:
Very personal opinion but this comment doesn't help me much. The code suggests this is a default prop for `material-symbols-outlined`, the comment just reiterates that

Totally fine for it to stick around